### DISCUSSION TITLE
Discord Integration😉

### DISCUSSION THREAD

**Discussion by Insane-alastor1** [namidaco/namida#418] [Ideas]
_First of All Thanks For Creating Such a Wonderful Handy App ❤️‍🩹_

**[Discord Rich Presence](https://discord.com/developers/docs/rich-presence/overview)✨** 

- Having only a **Single button** with Spotify Link to the song
- Able to Change the **Status(🌙Idle/⛔DND)**
- Make it Listening Through Namida if Possible🙌

No need to make is u don't want to as it's kinda like an Offline player so...

**Comment by VerySweetBread** [comment]
#114

**Reply by Insane-alastor1** [reply]
Tnx😌

**Comment by MSOB7YY** [comment]
thank u for the awesome issue and spirit, not in near plans currently but u can use https://github.com/dead8309/Kizzy as alternative

altho there won't be any links to spotify. maybe to yt only.
